doc/doxygen/src/build-system-basics.md

```diff
@@ -55,6 +55,11 @@ For a `FEATURE` to be provided by a `board` it must meet 2 criteria, and for
 - `FEATURES_OPTIONAL` are "nice to have" `FEATURES`, not needed but useful. If
   available they are always included.
 
+- `FEATURES_OPTIONAL_ALT` are nice to have `FEATURES` that fullfil a similar
```

Comment:
```suggestion
- `FEATURES_OPTIONAL_ALT` are nice to have `FEATURES` that fulfill a similar
```

Also, why not naming it `FEATURES_OPTIONAL_ANY`, as it behaves to `FEATURES_REQUIRED_ANY` the same way as `FEATURES_OPTIONAL` behave to `FEATURES_REQUIRED`?

Reply:
Both are correct, `fullfil` is the british english.

Reply:
The name change makes sense though. Will do.